plugins/restart.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha1
+kind: Plugin
+metadata:
+  name: "restart"
+spec:
+  platforms:
+  - uri: https://github.com/achanda/kubectl-restart/archive/v0.0.2.zip
+    sha256: 2dd5394555f645dbfb1c3f65d017356912ee59f52c3d6d71364440ac44780e8b
+    files:
+    - from: "/kubectl-restart-*/*"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+  version: "v0.0.1"
+  shortDescription: "Restarts a pod with the given name"
+  description: |
+    Restarts a pod with the given name. The existing pod
+    will be deleted and created again, not a true restart.
+  flags:
+    - name : "namespace"
```

Comment:
You should not define this flag. kubectl will provide this to you as an environment variable, when called like `kubectl plugin -n=FOO restart`. 

See [docs](https://kubernetes.io/docs/tasks/extend-kubectl/kubectl-plugins/):

> `KUBECTL_PLUGINS_CURRENT_NAMESPACE`: The current namespace that is the context for this call. This is the actual namespace to be used, meaning it was already processed in terms of the precedence between what was provided through the kubeconfig, the --namespace global flag, environment variables, and so on.

This way, you don't need to handle `-n` vs `--namespace` and its position etc. Kubectl is supposed to do this for you.